### PR TITLE
HOTFIX: fixes soft failing on settings/profile routes

### PR DESCRIPTION
**Pull request by alishaz-polymath**
## What does this PR do?

Hotfix for soft failing kbar routes in `settings/profile`

Fixes error shown in [LOOM](https://www.loom.com/share/cba7e02134dd4652b9119cec1ba2febf)

**Environment**: Staging(main branch)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- [x] Try using KBar for routing to Settings/Profile related locations

### CODE REVIEW

File: packages/app-store/apps.browser.generated.tsx [hunk 1]
```diff
@@ -100,4 +100,4 @@ whereby:whereby_installAppButton,
 wipemycalother:wipemycalother_installAppButton,
 zapier:zapier_installAppButton,
 zoomvideo:zoomvideo_installAppButton,
-};
\ No newline at end of file
+};
```
Comment (alishaz-polymath) [inline packages/app-store/apps.browser.generated.tsx]: No idea why this is showing up. Probably some windows related non-sense

File: packages/app-store/apps.server.generated.ts [hunk 2]
```diff
@@ -49,4 +49,4 @@ whereby:whereby_api,
 wipemycalother:wipemycalother_api,
 zapier:zapier_api,
 zoomvideo:zoomvideo_api,
-};
\ No newline at end of file
+};
```
Comment (alishaz-polymath) [inline packages/app-store/apps.server.generated.ts]: No idea why this is showing up. Probably some windows related non-sense (again)

File: apps/web/components/Kbar.tsx [hunk 3]
```diff
@@ -106,31 +106,31 @@ export const KBarRoot = ({ children }: { children: React.ReactNode }) => {
       section: "Profile Settings",
       shortcut: ["p", "s"],
       keywords: "setting profile",
-      perform: () => router.push("/settings"),
+      perform: () => router.push("/settings/profile"),
```
Comment (alishaz-polymath) [inline apps/web/components/Kbar.tsx:109]: Directly going to `settings/profile` is working but if we put `settings` alone, it looks to access user property before it is accessible. This works as a hotfix 🤷

**Review by alishaz-polymath — COMMENTED** [review]
Self review added

**Review by hariombalhara — APPROVED** [review]

**Review by PeerRich — APPROVED** [review]

**Pull request merged**